src/Menu/Menu.js

```diff
@@ -72,8 +66,28 @@ export default function Menu({
   );
 }
 
+const isGroupedData = (data: Items | ItemGroups) => {
+  return data[0].hasOwnProperty('items');
+};
+
+const groupifyData = (data: Items | ItemGroups) => {
+  return isGroupedData(data) ? data : [{ items: data }];
+};
+
+export const getItems = (data: Items | ItemGroups) => {
+  // $FlowFixMe https://github.com/facebook/flow/issues/5885
+  const itemGroups: ItemGroups = groupifyData(data);
+  return itemGroups.reduce((acc, group) => {
+    return group.items && group.items.length
```

Comment:
Should we add a guard for group.items.filter?

Reply:
Filter is a method on array.prototype so it should not need guarded. We already check for items and items.length and items is flow typed to be an array, so I think it’s pretty safe.